pkgs/development/python-modules/cmigemo/default.nix

```diff
@@ -0,0 +1,36 @@
+{ lib, buildPythonPackage, fetchPypi, six, cmigemo, pytest }:
+
+buildPythonPackage rec {
+  pname = "cmigemo";
+  version = "0.1.6";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "09j68kvcskav2cqb7pj12caksmj4wh2lhjp0csq00xpn0wqal4vk";
+  };
+
+  propagatedBuildInputs = [ six cmigemo ];
```

Comment:
Is the python package linked against cmigemo or does it requires an executable from it?

Reply:
It does not require an executable from cmigemo. Removed from propagated build inputs:
https://github.com/NixOS/nixpkgs/pull/115729/commits/7d30cd9705e072431eb08c323f4b436dc3d7b2f0#diff-d5dc448c3c8c0ff51881e3516fc05939ac863c6e8a641dfb8c06ab3fb31a3699R12